apps/web/src/hooks/useIsSafenetEnabled.ts

```diff
@@ -6,6 +7,10 @@ const useIsSafenetEnabled = () => {
   const { safe } = useSafeInfo()
   const { data: safenetConfig } = useGetSafenetConfigQuery()
```

Comment:
I think we should pass the `skipToken` here in case safenet is not enabled such that RTK query skips the request.